test/Test/DupIO/Conduit/Closed.hs

```diff
@@ -17,13 +17,13 @@ import Test.Util.TestSetup
 
 tests :: TestTree
 tests = testGroup "Test.DupIO.Conduit.Closed" [
-      testLocalOOM "withoutDupIO.OOM" test_withoutDupIO
-    , testCaseInfo "innerDupIO.OK"    test_innerDupIO
+      testLocalOOM "sourceWithoutDupIO.OOM" test_sourceWithoutDupIO
+    , testCaseInfo "sourceInnerDupIO.OK"    test_sourceInnerDupIO
```

Comment:
@FinleyMcIlwaine I'm currently rebasing this PR on top of the latest master, so don't worry about fixing this, but: these are not really _sources_, are they? They use `Lift` only , not `Yield` (that's what makes them _closed_ after all). I guess I will rename them to `closedWithoutDupIO` and co.